org.otter_browser.OtterBrowser.json

```diff
@@ -0,0 +1,38 @@
+{
+  "app-id": "org.otter_browser.OtterBrowser",
+  "base": "io.qt.qtwebkit.BaseApp",
+  "base-version": "stable",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "otter-browser",
+  "rename-desktop-file": "otter-browser.desktop",
+  "rename-appdata-file": "otter-browser.appdata.xml",
+  "rename-icon": "otter-browser",
+  "finish-args": [
+    "--device=dri",
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--filesystem=xdg-download",
+    "--env=QTWEBENGINE_DISABLE_SANDBOX=1"
```

Comment:
So does it use webengine or webkit?

Reply:
Can do both, you can choose, but WebEngine is more experimental.

Reply:
Disabling sandboxing sounds worse than not having qtwebengine enabled at all.

Reply:
Well it doesn’t run in Flatpak and we have Bubblewrap anyway, I don’t see a problem with this, I took this from existing Flathub manifests.

Reply:
@barthalion Revived QtWebKit is quite capable. No QtWebEngine isn't bad at all really. ;-)

Reply:
I dunno, keep sandbox disabled or disable WebEngine?